lib/install/esbuild.js

```diff
@@ -0,0 +1,15 @@
+const watch = process.argv.includes("--watch") && {
+  onRebuild(error) {
+    if (error) console.error("[watch] build failed", error);
+    else console.log("[watch] build finished");
+  },
+};
+
+require("esbuild")
+  .build({
+    entryPoints: ["app/javascript/application.js"],
+    bundle: true,
+    outfile: "app/assets/esbuilds/application.js",
```

Comment:
Should this be `outdir: "app'assets/esbuilds"` instead? That would allow multiple `entryPoints` out of the box.

Example:

```
entryPoints: ["app/javascript/application.js", "app/javascript/admin.js"],
outdir: "app/assets/esbuilds"
```

Reply:
It would also prevents typos and follow naming conventions.